bin/metrics-http-json-deep.rb

```diff
@@ -85,7 +85,7 @@ def deep_value(hash, scheme = '')
       if value.is_a?(Hash)
         deep_value(value, "#{scheme}.#{ekey}")
       else
-        output "#{scheme}.#{ekey}", value unless config[:numonly] && !value.is_a?(Integer)
+        output "#{scheme}.#{ekey}", value unless config[:numonly] && !value.is_a?(Numeric)
```

Comment:
hmm, if we are gonna allow floats lets cap the number of decimals places, even better would be to have it make it a function param with a sane default.

Reply:
Could you please explain what exactly do you mean by a 'a function param'?
Does it mean changing the --number option from a flag to argument parameter (how is it defined?) and take the decimal places allowance? 
btw, I'm new to Ruby...

Reply:
sorry I must have missed this but it looks like you figured out what I had in mind. Ping me once the CI issues have been fixed (or if you need help resolving them).

Reply:
I actually need help with resolving them, as the log doesn't supply useful information for understanding what's going on in the build. Thanks!

Reply:
So taking a look [here](https://travis-ci.org/sensu-plugins/sensu-plugins-http/jobs/413943718#L1814-L1815) it looks like its not related to your changes as you have not updated the dependencies. I will try to find some time to dig through the dependencies and see where `net-telnet` is coming from and lock the right dependency to ensure compatibility for ruby versions `< 2.3` and then will make a major release removing support for ruby `< 2.3` as its EOL and per [our policy](https://github.com/sensu/sensu-docs/blob/master/content/plugins/1.0/faq.md#what-is-the-policy-on-supporting-end-of-lifeeol-ruby-versions) we attempt to support non EOL versions of ruby.

I will figure out how to get this over the line.